internal/signalfx-agent/.golangci.yml

```diff
@@ -142,3 +142,10 @@ linters-settings:
 run:
   modules-download-mode: readonly
   timeout: 5m
+
+  # which files to skip: they will be analyzed, but issues from them
+  # won't be reported. Default value is empty list, but there is
+  # no need to include all autogenerated files, we confidently recognize
+  # autogenerated files. If it's not please let us know.
+  skip-files:
+    - pkg/monitors/prometheusexporter/conversion.go
```

Comment:
I don't think we want to skip the file entirely, but would use `// nolint: <linter>(,<linter>)` comments to disable checks on particular lines/blocks: https://golangci-lint.run/usage/false-positives/#nolint-directive. Did you run into an issue with this?

Reply:
I agree.